src/mmw/apps/geoprocessing_api/views.py

```diff
@@ -414,14 +412,8 @@ def start_analyze_land(request, format=None):
 
     geop_input = {'polygon': [area_of_interest]}
 
-    aoi_geom = GEOSGeometry(area_of_interest, srid=4326)
-    geop_task = 'nlcd'
```

Comment:
Is the `nlcd` processing tasks used anywhere else? Since we are always only using `nlcd_ara`, does it make sense to remove the `nlcd` entry from `src/mmw/mmw/settings/base.py` and any other related places?

Reply:
`base.py` was the only place it was used. Removed in 2ffadbf6. Good call.